### PR TITLE
[web] 120 add reg fields

### PR DESCRIPTION
**Pull request by elliotmessi**

### CODE REVIEW

File: ymir/web/src/services/user.js [hunk 1]
```diff
@@ -6,13 +6,15 @@ function sha1(value) {
...
-export function signup({ email, username, password, phone = null }) {
+export function signup({ email, username, password, phone = null, organization, scence }) {
   password = sha1(password)
   return request.post("/users/", {
     email,
     username,
     password,
     phone,
+    organization,
+    scence,
```
Comment (IJtLJZ8Rm4Yr) [inline ymir/web/src/services/user.js]: scene

Comment (elliotmessi) [reply ymir/web/src/services/user.js]: resolved

File: ymir/web/src/locales/modules/signup.ts [hunk 2]
```diff
@@ -22,6 +22,10 @@ const signup = {
   "signup.phone.placeholder": { cn: "请输入手机号码", en: "Please input your phone", },
   "signup.pwd.placeholder": { cn: "请输入密码", en: "Please input your password", },
   "signup.repwd.placeholder": { cn: "请再次输入密码", en: "Please input your password again", },
+  "signup.org": { cn: "所属机构", en: "Organization", },
+  "signup.org.placeholder": { cn: "请输入您所在的机构/学校名称", en: "Please input school/org. your worked for", },
+  "signup.scence": { cn: "应用场景", en: "Usage Scence", },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/signup.ts]: s/scence/scene

File: ymir/web/src/locales/modules/signup.ts [hunk 3]
```diff
@@ -22,6 +22,10 @@ const signup = {
...
+  "signup.scence.placeholder": { cn: "请输入你的具体应用场景描述", en: "Please describe usage scence", },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/signup.ts]: s/scence/scene

File: ymir/web/src/pages/user/signup/index.js [hunk 4]
```diff
@@ -113,6 +115,18 @@ const Signup = ({ signupApi, loginApi, history }) => {
...
                 <Input allowClear placeholder={t('signup.phone.placeholder')} prefix={<SmartphoneIcon style={{ color: 'rgba(0, 0, 0, 0.45)'}} />} />
               </Form.Item>
+              <Form.Item
+                label={t("signup.org")}
+                name="organization"
+              >
+                <Input allowClear placeholder={t('signup.org.placeholder')} prefix={<SmartphoneIcon style={{ color: 'rgba(0, 0, 0, 0.45)'}} />} />
+              </Form.Item>
+              <Form.Item
+                label={t("signup.scence")}
```
Comment (phoenix-xhuang) [inline ymir/web/src/pages/user/signup/index.js]: s/scence/scene

File: ymir/web/src/pages/user/signup/index.js [hunk 5]
```diff
@@ -113,6 +115,18 @@ const Signup = ({ signupApi, loginApi, history }) => {
...
+                name="scene"
+              >
+                <Input allowClear placeholder={t('signup.scence.placeholder')} prefix={<SmartphoneIcon style={{ color: 'rgba(0, 0, 0, 0.45)'}} />} />
```
Comment (phoenix-xhuang) [inline ymir/web/src/pages/user/signup/index.js]: s/scence/scene

Comment (elliotmessi) [reply ymir/web/src/pages/user/signup/index.js]: fixed

File: ymir/web/src/pages/user/signup/index.js (hunk 4 above)
Comment (elliotmessi) [reply ymir/web/src/pages/user/signup/index.js]: fixed

File: ymir/web/src/locales/modules/signup.ts (hunk 3 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/signup.ts]: fixed

File: ymir/web/src/locales/modules/signup.ts (hunk 2 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/signup.ts]: fixed

**Review by phoenix-xhuang — APPROVED** [review]

**Review by fenrir-z — APPROVED** [review]

**Pull request merged**